lib/pages/game_form.dart

```diff
@@ -224,7 +226,7 @@ class _GameFormPageState extends State<GameFormPage> {
                         locationTitle: _locationTitle,
                         positionsRequired: [],
                         registeredUsers: [],
-                      ));
+                      ), userId);
```

Comment:
Why not set it on the properties of SportEvent? 
Like
```dart
SportEvent(
  sportId: widget.sportId,
  userId: userId)
```

Reply:
resolved